Release/tests/functional/http/client/outside_tests.cpp

```diff
@@ -63,6 +63,27 @@ TEST_FIXTURE(uri_address, outside_cnn_dot_com)
     });
 }
 
+TEST_FIXTURE(uri_address, outside_wikipedia_compressed_http_response)
```

Comment:
Changed this to wiki instead of google. Reason being that google is redirecting (maybe because I'm writing this from Europe with a different TLD) with a 302. That is handled fine on Windows (inside WinHTTP) but not on OSX (requires application level code to handle the redirect). Wiki isn't redirecting so it's a better test url. 

I will add a separate PR to fix up other tests that have similar issues.